code/modules/hydroponics/hydroponics.dm

```diff
@@ -252,10 +252,9 @@
 
 //This is where stability mutations exist now.
 			switch(myseed.instability)
-				if(80 to 100)
-					if(prob(20))
-						mutate(0, 0, 0, 0, 0, 0, 0, 10, 0) //Exceedingly low odds of gaining a trait.
-				if(60 to 79)
+				if(100 to 80)
+					mutate(0, 0, 0, 0, 0, 0, 0, 5, 0) //Exceedingly low odds of gaining a trait.

```

Comment:
was it intentional to double the probability for mutation? I don't see it documented.

if you were trying to make them the same, you're looking for
```suggestion
					mutate(0, 0, 0, 0, 0, 0, 0, 2, 0) //Exceedingly low odds of gaining a trait.
```